.rubocop.yml

```diff
@@ -74,10 +75,10 @@ Style/IndentationWidth:
 Style/IndentHash:
   Severity: fatal
   Enabled: true
+  EnforcedStyle: consistent
 
 Style/InlineComment:
-  Severity: fatal
-  Enabled: true
+  Enabled: false
```

Comment:
We focused on having specs that documented the code being written, I stand by @Integralist in saying that comments are a code smell.